mcp/shared.go

```diff
@@ -275,6 +275,8 @@ func sessionMethod[S Session, P Params, R Result](f func(S, context.Context, P)
 
 // Error codes
 const (
+	// TODO: should these be unexported?
```

Comment:
I don't recall why they were exported, but we can certainly unexport them for now.